airbyte-integrations/connectors/source-google-adwords-singer/build.gradle

```diff
@@ -21,10 +21,3 @@ task('installSingerTap', type: PythonTask) {
     command = "install tap-adwords==1.12.0"
```

Comment:
lets move `tap-adwords==1.12.0` to `setup.py` and remove this task (`installSingerTap`)

Reply:
OK

Reply:
@keu I can't run tests with this in setup.py. Also it was specified in Dokerfile before me. The problem is in pip install requirements.txt it fails with timeout error

Reply:
ok, lets keep it as it is for now